src/main/resources/com/atlassian/performance/tools/report/chart/per-action-chart-template.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>JPT latency chart</title>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.13.0/moment.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.3/Chart.bundle.min.js"></script>
+</head>
+<body>
+<div>
+<canvas id="chart"></canvas>
+</div>
+<div id="labelButtons">
+</div>
+<script>
+
+    var labelChooserListener = function (event) {
+        var label = event.target.innerText;
+        var labelIndex = chartData.labels.findIndex(v => v === label);
+
+        if ( labelIndex >= 0) {
+            chartData.labels.splice(labelIndex, 1);
+
+            for (var index = 0; index < chartData.datasets.length; ++index) {
+                chartData.datasets[index].data.splice(labelIndex, 1);
+            }
+
+        } else {
+
+        }
+
+        chart.update();
+        event.target.remove()
+    };
+
+    var buttons = document.getElementById("labelButtons");
+
+    var chartData = '<%= data =%>';
+
+    chartData.labels.forEach(function(element) {
+        var text = document.createTextNode(element);
+        var button = document.createElement("button");
+        button.append(text);
+        buttons.append(button);
+        button.addEventListener('click', labelChooserListener);
+    });
+
+    var chart = new Chart(document.getElementById("chart").getContext("2d"), {
+        type: 'horizontalBar',
+        data: chartData,
+        options: {
+            title: {
+                display: true,
+                text: "Latency comparison across for different actions"
+            },
+            scales: {
+                xAxes: [{
+                    scaleLabel: {
+                        display: true,
+                        labelString: "Duration [ms]"
+                    },
+                    ticks: {
+                        beginAtZero:true
+                    }
+                }],
+                yAxes: [{
+                    scaleLabel: {
+                        display: true,
+                        labelString: "Actions"
```

Comment:
I often find myself explaining `time-series-chart.html` and the unit-less series are most often misinterpreted.
Is it total action count? Then let's:
```suggestion
                        labelString: "Actions [#]"
```
Is it action count per minute? Then let's:
```suggestion
                        labelString: "Actions [# / min]"
```

Reply:
Those are Action types.
<img width="367" alt="screenshot 2019-02-25 at 13 21 07" src="https://user-images.githubusercontent.com/129621/53337002-6348a800-3900-11e9-9e27-7bf854fa311d.png">